src/main/index.ts

```diff
@@ -27,14 +27,18 @@ process.env.VENCORD_USER_DATA_DIR = DATA_DIR;
 function init() {
     const { disableSmoothScroll, hardwareAcceleration } = Settings.store;
 
-    if (hardwareAcceleration === false) app.disableHardwareAcceleration();
+    if (hardwareAcceleration === false) {
+        app.disableHardwareAcceleration();
+    } else {
```

Comment:
probably best to only do this on linux?
```suggestion
    } else if (process.platform === "linux") {
```